kernel-extractor.py

```diff
@@ -0,0 +1,10 @@
+import json
+from jupyter_client.kernelspec import KernelSpecManager
+
+specs = KernelSpecManager().get_all_specs()
+result = {}
+for key, spec in specs.items():
+    result[key] = spec
+
+result = json.dumps(result)
```

Comment:
I think we can turn this into

```suggestion
result = json.dumps(KernelSpecManager().get_all_specs())
```